csharp/ColumnWriter.cs

```diff
@@ -159,7 +159,25 @@ public LogicalColumnWriter<TElement> LogicalWriterOverride<TElement>(int bufferL
         protected static extern unsafe IntPtr TypedColumnWriter_WriteBatchSpaced_FixedLenByteArray(
             IntPtr columnWriter, long numValues, short* defLevels, short* repLevels, byte* validBits, long validBitsOffset, FixedLenByteArray* values);
 
-        protected readonly IntPtr Handle;
+        protected IntPtr Handle
```

Comment:
I don't believe we need to worry about binary compatibility here, because `Handle` is a protected member and the `ColumnWriter` and `ColumnWriter<TValue>` constructors are both internal, so it shouldn't be possible for code outside of ParquetSharp to be accessing this member.